src/extension.ts

```diff
@@ -145,8 +150,8 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
         registerCommand('pygls.server.executeCommand', async () => {
             await executeServerCommand(lsClient);
         }),
-        registerCommand('kedro.sendDefinitionRequest', async () => {
-            await executeServerDefinitionCommand(lsClient);
+        registerCommand('kedro.sendDefinitionRequest', async (word) => {
+            await executeServerDefinitionCommand(lsClient, word);
```

Comment:
The idea is that the webview use vscode command directly, instead of importing any function. 

The command itself would have access of lsClient already. This enable restart LSP without disconnecting the webview from it.